presentation-core/src/main/java/tachiyomi/presentation/core/util/Modifier.kt

```diff
@@ -53,8 +55,17 @@ fun Modifier.clickableNoIndication(
 fun Modifier.runOnEnterKeyPressed(action: () -> Unit): Modifier = this.onPreviewKeyEvent {
     when (it.key) {
         Key.Enter, Key.NumPadEnter -> {
-            action()
-            true
+            // Physical keyboards generate two event types:
+            // - KeyDown when the key is pressed
+            // - KeyUp when the key is released
+            // To match the behavior of software keyboards (IME),
+            // trigger [action] only on the KeyUp event.
+            if (it.type == KeyEventType.KeyUp) {
```

Comment:
But we're working with physical keyboard here where stuff usually triggers on `KeyDown`

Reply:
shouldn't the behavior be uniform across software and physical keyboard?